### ISSUE TITLE
Hue Labs not working

### ISSUE THREAD

**Issue by oleg2204** [mariusmotea/diyHue#146]
I just noticed that the Feature "Hue Labs" in the official Hue App is not working. 
It searches for the bridge but doesnt find anything. Even when i type in the IP it still cant connect.

Is there some fix? Someone already took a look at "Hue Entertainment"??

**Comment by mariusmotea** [comment]
Currently in a business trip, i will take a look when i will arrive home. Is possible that this is using remote api.

**Comment by mariusmotea** [comment]
I test now, when is displaying "Search for Hue Bridges" there are no request over the local network, so very likely is trying to communicate with the bridge via cloud services/remote api.

**Comment by oleg2204** [comment]
I have made some research regarding the remote api of philips.  I think we could get the whole thing working.

There exists a way to get root access on the Hue Bridge. (google)
The communication to the philips servers uses HTTP but encrypts the content with AES. The AES key is hardcoded in the firmware on the bridge. 
If someone can get the AES key the whole thing could be decrypted and reverse engineered.

We would need someone who owns a hue bridge and who can perform the root hack. Than one must find the AES key and record the traffic. Than decrypt the whole thing and reconstruct the protocol.

Problem is: I dont own a hue bridge and i dont have the needed knowledge for such a operation :(  Also time is rare.

**Comment by mariusmotea** [comment]
I was sure the traffic is encrypted. I also don't own a hue bridge. Once access is in place i believe is easy to retrieve the key, but there is a risk to brick the device while doing this.

**Comment by cheesemarathon** [comment]
@mariusmotea Did I read in another issue that you managed to root a hue bridge? Can we retrieve the required key?

**Comment by cheesemarathon** [comment]
Closing as hue remote API required for support. Discussions regarding supporting the remote API will take place in Slack.
